lib/pact_broker/pacts/merger.rb

```diff
@@ -32,21 +43,15 @@ def merge_pacts original_json, additional_json
 
       private
 
-      def matching_request? original_interaction, new_interaction
-        same_description_and_state?(original_interaction, new_interaction) &&
-          same_request_properties?(original_interaction["request"], new_interaction["request"])
-      end
-
       def same_description_and_state? original, additional
         original["description"] == additional["description"] &&
           original["provider_state"] == additional["provider_state"]
       end
 
       def same_request_properties? original, additional
-        method_matches = original["method"] == additional["method"]
-        path_matches = original["path"] == additional["path"]
-
-        method_matches && path_matches && original["headers"] == additional["headers"]
+        attributes_match = %w(method path query body headers).all? do |attribute|
```

Comment:
assigning this to a variable is fairly pointless but doesn't hurt anything - thoughts on keeping vs. removing?